internal/pkg/config/v1/mock.go

```diff
@@ -114,7 +114,7 @@ func APICredentialConfigMock() *Config {
 
 func UnauthenticatedCloudConfigMock() *Config {
 	c := AuthenticatedCloudConfigMock()
-	c.Contexts = nil
+	c.CurrentContext = ""
```

Comment:
logging out doesn't delete all contexts. setting it to nil will cause contexts mapping error.